source/jormungandr/tests/kirin_realtime_tests.py

```diff
@@ -1757,6 +1757,194 @@ def test_add_new_trip(self):
+        resp, status = self.query_region("commercial_modes/commercial_mode:additional_service", check=False)
+        assert status == 404
+        assert resp['error']['message'] == 'ptref : Filters: Unable to find object'
+
+        # New disruption, a new trip with 2 stop_times in realtime
+        self.send_mock(
+            "additional-trip",
+            "20120614",
+            "added",
+            [
+                UpdatedStopTime(
+                    "stop_point:stopC",
+                    arrival_delay=0,
+                    departure_delay=0,
+                    is_added=True,
+                    arrival=tstamp("20120614T080100"),
+                    departure=tstamp("20120614T080100"),
+                    message='on time',
+                ),
+                UpdatedStopTime(
+                    "stop_point:stopB",
+                    arrival_delay=0,
+                    departure_delay=0,
+                    is_added=True,
+                    arrival=tstamp("20120614T080102"),
+                    departure=tstamp("20120614T080102"),
+                ),
+            ],
+            disruption_id="new_trip",
+            effect="additional_service",
+            physical_mode_id="physical_mode:Bus",  # this physical mode exists in kraken
+        )
+
+        # Check new disruption 'additional-trip' to add a new trip
+        disruptions_after = self.query_region(disruption_query)
+        assert nb_disruptions_before + 1 == len(disruptions_after['disruptions'])
+
+        # Verify that network, line, commercial_mode of the new trip are present
+        resp = self.query_region("networks/network:additional_service")
+        assert "networks" in resp
+        resp = self.query_region("lines/line:stopC_stopB")
+        assert "lines" in resp
+        resp = self.query_region("commercial_modes/commercial_mode:additional_service")
+        assert "commercial_modes" in resp
+
+        resp = self.query_region("networks/network:additional_service/physical_modes")
+        assert resp["physical_modes"][0]["id"] == "physical_mode:Bus"
+
+        resp = self.query_region("physical_modes/physical_mode:Bus/networks")
+        networks = [nw["id"] for nw in resp["networks"]]
+        assert "network:additional_service" in networks
+
+        # network by line should work
+        resp = self.query_region("lines/line:stopC_stopB/networks")
+        assert resp["networks"][0]["id"] == "network:additional_service"
+
+        # The physical_mode sent in gtfs-rt should be present in the new line added
+        resp = self.query_region("lines/line:stopC_stopB/physical_modes")
+        assert resp["physical_modes"][0]["id"] == "physical_mode:Bus"
+
+        # The default commercial_mode used for a new line should be present
+        resp = self.query_region("lines/line:stopC_stopB/commercial_modes")
+        assert resp["commercial_modes"][0]["id"] == "commercial_mode:additional_service"
```

Comment:
I'm maybe wrong, but shouldn't we be able to set the commercial mode when sending the disruption ? 
Do we always attach all new trips to this default `"commercial_mode:additional_service"`mode ?